src/main/java/io/pivotal/cfapp/controller/DbmsOnlyPoliciesController.java

```diff
@@ -43,6 +43,12 @@ public DbmsOnlyPoliciesController(
                 .map(ResponseEntity::ok);
     }
 
+    @DeleteMapping("/policies/message/{id}")
```

Comment:
Not convinced on the policy `type` = `message` for this policy. We need to find a better type name.

Reply:
Likely `message` should be renamed to `label`.

Reply:
Changed the name to ResourceNotificationPolicy